config/lib/authorisation.js

```diff
@@ -24,28 +23,29 @@ exports.isLoggedIn = function (req, res, next) {
  * @param {Object} user - The req.user object.
  * @returns {boolean}
  */
-function hasAllRights(names, user) {
-	var hasRights = true;
+const hasAllRights = function (names, user) {
+	let hasRights = true;
 	names.forEach(function (name) {
 		hasRights &= UserRights.userHasRights(user, name);
 	});
 	return hasRights;
-}
+};
 
 /**
  * A custom function that checks whether the user is authorised to perform a request (/access a webpage).
  * It is build on top of the Passport library @see {@link http://passportjs.org/}
+ * If we are in the testing environment no authorisation is performed.
  *
```

Comment:
maybe add development mode in addition to testing here?

Reply:
I thought about that as well, but that would make us unable to test whether the user rights and login are properly working in the development environment. So I am still a bit doubtful about adding it.

Reply:
I thought about that, but that would mean we can not actually test whether the authorisation and authentication system is working properly in development mode. In testing mode it allows us to easily test the API without having to set user rights and all that. So I am not entirely sure whether I should add that. Maybe a better alternative is to add the option to set some environment variable which allows you to skip login and user rights.